integration-cli/docker_cli_run_test.go

```diff
@@ -3092,11 +3342,16 @@ func (s *DockerSuite) TestRunLoopbackOnlyExistsWhenNetworkingDisabled(c *check.C
 
 // Issue #4681
 func (s *DockerSuite) TestRunLoopbackWhenNetworkDisabled(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	dockerCmd(c, "run", "--net=none", "busybox", "ping", "-c", "1", "127.0.0.1")
+	//testRequires(c, DaemonIsLinux)
```

Comment:
This test is funny...
There is another test `TestRunContainerNetwork` which tests the exact same thing, but without `--net=none`... I think the other test is probably not valid... just taking a note of this.
ping @mavenugo.